injectable_generator/pubspec.yaml

```diff
@@ -1,26 +1,26 @@
 name: injectable_generator
 description: Injectable is a convenient code generator for get_it. Inspired by Angular DI, Guice DI and inject.dart.
-version: 2.6.2
+version: 2.6.3
 homepage: https://github.com/Milad-Akarie/injectable
 
 environment:
   sdk: ">=3.0.0<4.0.0"
 
 dependencies:
-  build: ^2.4.1
-  source_gen: ^1.5.0
-  path: ^1.9.0
+  build: ^2.4.2
+  source_gen: ^2.0.0
+  path: ^1.9.1
   glob: ^2.1.2
-  analyzer: ">=6.4.1 <7.0.0"
-  code_builder: ^4.10.0
-  dart_style: ^2.3.6
-  injectable: ^2.4.3
-#    path: ../injectable
-  collection: ^1.17.1
+  analyzer: ^7.1.0
+  code_builder: ^4.10.1
+  dart_style: ^3.0.1
+  injectable: ^2.5.1
```

Comment:
`injectable: ^2.5.1` does not correspond to a valid injectable version
Can you double-check this?